Formula/forgit.rb

```diff
@@ -0,0 +1,28 @@
+class Forgit < Formula
+  desc "Interactive git commands in the terminal"
+  homepage "https://github.com/wfxr/forgit"
+  url "https://github.com/wfxr/forgit/releases/download/23.04.0/forgit-23.04.0.tar.gz"
+  sha256 "89826d12b551829c4bce396551b1505c0604ba29130f4b8e2f842c2ca0b4c9e3"
+  license "MIT"
+
+  depends_on "fzf"
+
+  def install
+    bin.install "bin/git-forgit"
+    bash_completion.install "completions/git-forgit.bash"
+    prefix.install "forgit.plugin.zsh"
+  end
+
+  def post_install
+    ohai "NOTE: forgit can be used as subcommand of git (e.g. 'git forgit log')."
+    ohai "If you like to have zsh/bash shell aliases (e.g. 'glo'), source the shell plugin:"
+    ohai "  source #{prefix}/forgit.plugin.zsh"
+    ohai "Add the line to your .zshrc/.bashrc to make this permanent."
+  end
```

Comment:
```suggestion
  def caveats
    <<~EOS
      A shell plugin has been installed at
        #{opt_pkgshare}/forgit.plugin.zsh
        #{opt_pkgshare}/forgit.plugin.sh
    EOS
  end
```

Reply:
@carlocab Good idea, thanks a lot. I updated the branch accordingly.